### ISSUE TITLE
Is continuity information in the normalized Patent Object Model?

### ISSUE THREAD

**Issue by mikessut** [USPTO/PatentPublicData#109]
It is my understanding that continuity information is contained within the bulk files for applications.  Does this information make it into the normalized Patent Object Model?

**Comment by bgfeldm** [comment]
Yes, within the Redbook us-related-documents, PAP continuity-data and SGML B600 are parsed and stored as relationIds, which also holds other related ids (regional and national filing ids), they can be retrieved with Patent getRelationIds(). For each DocumentId use getType() to return the DocumentIdType, such as ADDITION, DIVISION, CONTINUATION, and CONTINUATION_IN_PART.

**Comment by mikessut** [comment]
Thank you. Very helpful.

Does it make sense to output the types for the reatedIds in the json serializer?  Something like what I've done here?
https://github.com/mikessut/PatentPublicData/tree/mikessut/continuity
